### DISCUSSION TITLE
How can we use functions and commands in jitsi react-native sdk?

### DISCUSSION THREAD

**Discussion by amarnathduttaroy** [community/community#118515] [Programming Help]
How can we use functions and commands in JITSI MEET react-native SDK?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @amarnathduttaroy !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
